doc/user/content/sql/create-sink.md

```diff
@@ -42,9 +42,11 @@ Field | Use
 ------|-----
 **KAFKA BROKER** _host_ | The Kafka broker's host name without the security protocol, which is specified by the [`WITH` options](#with-options).) If you wish to specify multiple brokers (bootstrap servers) as an additional safeguard, use a comma-separated list. For example: `localhost:9092, localhost:9093`.
 **TOPIC** _topic&lowbar;prefix_ | The prefix used to generate the Kafka topic name to create and write to.
+**KEY (** _key&lowbar;column&lowbar;list_ **)** | An optional list of columns to use for the Kafka key. If unspecified, the Kafka key is left unset. {{< version-added v0.5.1 />}}
 **WITH OPTIONS (** _option&lowbar;_ **)** | Options affecting sink creation. For more details see [`WITH` options](#with-options).
+**CONSISTENCY TOPIC** _consistency&lowbar;topic_ | The Kafka topic name to write [consistency metadata](#consistency-metadata) to.
+**CONSISTENCY FORMAT** _format_ | The format of the Kafka consistency topic, defaults to Avro for Avro-encoded sinks.
```

Comment:
Can you add one of those `{{< version-added >}}` tags to the docs here (see `KEY` for an example)?